x/ccv/provider/client/proposal_handler.go

```diff
@@ -536,33 +440,8 @@ func postConsumerRemovalProposalHandlerFn(clientCtx client.Context) http.Handler
 	}
 }
 
-func postEquivocationProposalHandlerFn(clientCtx client.Context) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) {
-		var req EquivocationProposalReq
-		if !rest.ReadRESTReq(w, r, clientCtx.LegacyAmino, &req) {
-			return
-		}
-
-		req.BaseReq = req.BaseReq.Sanitize()
-		if !req.BaseReq.ValidateBasic(w) {
-			return
-		}
-
-		content := types.NewEquivocationProposal(req.Title, req.Description, req.Equivocations)
-
-		msg, err := govtypes.NewMsgSubmitProposal(content, req.Deposit, req.Proposer)
-		if rest.CheckBadRequestError(w, err) {
-			return
-		}
-
-		if rest.CheckBadRequestError(w, msg.ValidateBasic()) {
-			return
-		}
-
-		tx.WriteGeneratedTxResponse(clientCtx, w, req.BaseReq, msg)
-	}
-}
-
+func CheckPropUnbondingPeriod(clientCtx client.Context, propUnbondingPeriod time.Duration) {
```

Comment:
This seems like was added by accident and being part of a huge comment section. The `CheckPropUnbondingPeriod` function appears earlier from what I see.